Images/microsoft_powershell_centos7/PowerShellPackage.ps1

```diff
@@ -11,9 +11,10 @@ param (
     [string] $destination = '/mnt'
 )
 
-git clone --recursive https://github.com/$fork/powershell.git -b $branch 2> $null
+git clone --quiet https://github.com/$fork/powershell.git -b $branch
 Push-Location
 Set-Location "$location"
+git submodule update --init --recursive --quiet

```

Comment:
Can we use `git clone -q --recursive` to avoid `git submodule update --init --recursive --quiet`?